<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="groupBox_rad1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>groupBox1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupBox_rad2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>groupBox2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupBox_rad3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>groupBox3</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
